packages/slice-machine/package.json

```diff
@@ -12,19 +12,16 @@
     "node": ">=14"
   },
   "scripts": {
-    "dev": "concurrently \"next dev\" \"npm run dev --prefix ../../e2e-projects/next\"",
+    "build": "tsc --project scripts/tsconfig.json && rollup -c ./rollup-server.config.js && cross-env NODE_OPTIONS=--max_old_space_size=8192 next build",
+    "start": "concurrently --prefix '{name}' --names 'next-app,dev-server' --prefix-colors 'red,yellow' 'npm run start:dev' 'npm run start:dev-server'",
+    "start:dev": "concurrently --prefix 'none' 'next dev' 'npm run dev --prefix ../../e2e-projects/next'",
+    "start:dev-server": "cross-env PORT=9999 CWD=../../e2e-projects/next ENV=SM concurrently --prefix 'none' 'rollup -w -c ./rollup-server.config.js' 'nodemon -e ts,js --watch server --watch lib --ignore build/ build/server/index.js'",
     "test": "jest",
     "test:watch": "jest --watch --no-coverage",
-    "clean-build": "rimraf ./build/client",
     "postinstall": "node postinstall",
     "prepublishOnly": "npm run bundle",
-    "dev-server": "cross-env PORT=9999 CWD=../../e2e-projects/next ENV=SM concurrently \"rollup -w -c ./rollup-server.config.js\" \"nodemon -e ts,js --watch server --watch lib --ignore build/ build/server/index.js\"",
-    "build": "npm run build-scripts && npm run build-server && npm run build-next-app",
-    "build-scripts": "tsc --project scripts/tsconfig.json",
-    "build-server": "rollup -c ./rollup-server.config.js",
-    "build-next-app": "cross-env NODE_OPTIONS=--max_old_space_size=8192 next build",
-    "export-build": "next export && npm run clean-build",
-    "bundle": "npm run build && npm run export-build",
+    "bundle": "npm run build && next export && rimraf ./build/client",
+    "export-build": "next export && rimraf ./build/client",
```

Comment:
i don't think this is used anymore.

Reply:
It's used before the e2e tests in CI. Not sure if it's needed though?